src/main/java/org/opensearch/performanceanalyzer/reader/ReaderMetricsProcessor.java

```diff
@@ -60,11 +60,12 @@ public class ReaderMetricsProcessor implements Runnable {
     private NavigableMap<Long, OSMetricsSnapshot> osMetricsMap;
     private NavigableMap<Long, ShardRequestMetricsSnapshot> shardRqMetricsMap;
     private NavigableMap<Long, HttpRequestMetricsSnapshot> httpRqMetricsMap;
-    private NavigableMap<Long, MasterEventMetricsSnapshot> masterEventMetricsMap;
+    private NavigableMap<Long, ClusterManagerEventMetricsSnapshot> clusterManagerEventMetricsMap;
     private NavigableMap<Long, GarbageCollectorInfoSnapshot> gcInfoMap;
     private Map<AllMetrics.MetricName, NavigableMap<Long, MemoryDBSnapshot>> nodeMetricsMap;
     private NavigableMap<Long, FaultDetectionMetricsSnapshot> faultDetectionMetricsMap;
-    private NavigableMap<Long, MasterThrottlingMetricsSnapshot> masterThrottlingMetricsMap;
+    private NavigableMap<Long, ClusterManagerThrottlingMetricsSnapshot>
+            clusterManagerThrottlingMetricsMap;
```

Comment:
nit : can be a single line
```suggestion
    private NavigableMap<Long, ClusterManagerThrottlingMetricsSnapshot> clusterManagerThrottlingMetricsMap;
```